plugins/metrics/app/controllers/metrics/application_controller.rb

```diff
@@ -1,16 +1,118 @@
 # frozen_string_literal: true
-
 module Metrics
   class ApplicationController < DashboardController
+    helper_method :grafana_available
     authorization_context 'metrics'
     authorization_required
-
+    extend ErrorRenderer
```

Comment:
do we need that? I removed it and tested the error case and the error was rendered correctly

Reply:
removed it. not sure if needed or not